core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -90,23 +89,6 @@ public void testPorts() {
       }
   }
 
-  @Test
-  public void testRawDefaultValues() {
```

Comment:
@ctubbsii since I have added back the `VFS_CLASSLOADER_CACHE_DIR` I will add this test back